src/display/ui/default/eez/eez-flow.cpp

```
@@ -90,29 +86,26 @@ namespace eez {
 Assets *g_mainAssets;
 bool g_mainAssetsAreMutable;
 void fixOffsets(Assets *assets);
-bool decompressAssetsData(const uint8_t *assetsData, uint32_t assetsDataSize, Assets *decompressedAssets,
-                          uint32_t maxDecompressedAssetsSize, int *err) {
+bool decompressAssetsData(const uint8_t *assetsData, uint32_t assetsDataSize, Assets *decompressedAssets, uint32_t maxDecompressedAssetsSize, int *err) {
     EEZ_UNUSED(assetsData);
     EEZ_UNUSED(assetsDataSize);
     EEZ_UNUSED(decompressedAssets);
     EEZ_UNUSED(maxDecompressedAssetsSize);
     *err = -1;
```

Cppcheck (2.21.0)

[error] Null pointer dereference

(ctunullpointer)